### PR TITLE
Add controller helper as a standalone module

### PR DESCRIPTION
**Pull request by giusebar**
## Proposed changes

Please explain the changes you made. Add pictures or videos to explain them better if appropiate.

## Types of changes

What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

Put an x in the boxes that apply. You can also fill these out after creating the PR. This is a reminder of what we should look for before merging this code. I have:

- [ ] Read and follow the [contributing guidelines](https://github.com/shadow-robot/sr_documentation/blob/master/CONTRIBUTING.md).
- [ ] Checked that all tests pass with my changes
- [ ] Added tests (automatic or manual) that prove the fix is effective or that the feature works
- [ ] Added necessary documentation (if appropriate)
- [ ] Added the corresponding license to each file and add the current year of any one that you modified.
- [ ] Tested on real hardware (if appropriate)

### CODE REVIEW

File: sr_controllers_tools/src/sr_controllers_tools/sr_controller_helper.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Shadow Robot Company Ltd.
```
Comment (biofotis) [inline sr_controllers_tools/src/sr_controllers_tools/sr_controller_helper.py]: 2020

File: sr_controllers_tools/src/sr_controllers_tools/sr_controller_helper.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                    except rospy.ServiceException:
+                        success = False
+                    if not resp1.ok:
+                        success = False
+
+            switch_controllers = rospy.ServiceProxy(
+                'controller_manager/switch_controller', SwitchController)
+            try:
+                resp1 = switch_controllers(
+                    controllers_to_start, controllers_to_stop, SwitchControllerRequest.BEST_EFFORT)
```
Comment (biofotis) [inline sr_controllers_tools/src/sr_controllers_tools/sr_controller_helper.py]: this wont work now. You need to add 2 more parameters as the msg changed

File: sr_controllers_tools/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,203 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(sr_controllers_tools)
+
+## Compile as C++11, supported in ROS Kinetic and newer
```
Comment (biofotis) [inline sr_controllers_tools/CMakeLists.txt:4]: please remove comments to make it shorter

File: sr_controllers_tools/package.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>sr_controllers_tools</name>
+  <version>0.0.1</version>
+  <description>The sr_controllers_tools package</description>
+  <author email="software@shadowrobot.com">Shadow Robot Software Team</author>
```
Comment (biofotis) [inline sr_controllers_tools/package.xml]: i think author should be you

**Review by biofotis — DISMISSED** [review]

**Review by beatrizleon — APPROVED** [review]

**Pull request merged**